### PR TITLE
Add a new stage

### PR DESCRIPTION
**Pull request by ddysher**
**What this PR does / why we need it**:

Please merge https://github.com/caicloud/kube-ladder/pull/6 first. Only the second commit matters.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @bbbmj @ijumps 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ddysher** [comment]
@bbbmj @ijumps  please comment more features that you see fit.

File: README.md [hunk 1]
```diff
@@ -228,7 +230,101 @@ Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
...
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/)
+
+同时，Kubernetes 提供了 [FeatureGate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) 来控制不同的特性开关，我们可以通过 FeatureGate 来了解 Kubernetes 的新特性。此外，为了方便开发者和配置管理，Kubernetes 把所有配置都挪到了相对应的 GitHub 代码仓库中，即：
+- https://github.com/kubernetes/kube-scheduler
+- https://github.com/kubernetes/kube-controller-manager
+- https://github.com/kubernetes/kubelet
+- https://github.com/kubernetes/kube-proxy
+- https://github.com/kubernetes/kubectl
+
+当然，直接裸看配置有点硬核。为方便入手，下面我们简单总结部份功能（笼统的分为 Master 和 Node）：
```
Comment (bbbmj) [inline README.md]: 部份 -> 部分

**Comment by gaocegege** [comment]
Please rebase the master

File: README.md (hunk 1 above)
Comment (ddysher) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -227,7 +230,101 @@
 - 本阶段难度指数高，请合理调整你的心境。渡劫成功后，你对 Kubernetes 的掌握将会进入一个新的台（tian）阶（keng）。
 - 学习相关功能时，可以回顾其所在 SIG，看看能不能发现有用的资源。
 
-### 第五阶段 化神期（4-6 周，每周 8-10 小时）
+### 第五阶段 化神期（3-5 周，每周 6-8 小时）
```
Comment (bbbmj) [inline README.md]: 如 @ijumps 所说，这个东西应该是一个需知，而不是单独的一个阶段，也没有明确可做的目标。

是不是可以拎出来单独形成一个小文档

Comment (ddysher) [reply README.md]: learning from components is a great way to dig into Kubernetes features, and I want folks to know this as well.

**Pull request merged**